### PR TITLE
Added symbols needed by lib-imsdpl.so, Ant+ support, Pinner config, disable rfs_access

### PR DESCRIPTION
**Pull request by karthick111**
Signed-off-by: karthick111 <karthickrko61@gmail.com>

### CODE REVIEW

**Comment by karthick111** [comment]
@skritchz  @Shr3ps  can be merged

File: rootdir/Android.mk [hunk 1]
```diff
@@ -8,11 +8,21 @@ LOCAL_SRC_FILES    := etc/init.qcom.bt.sh
 include $(BUILD_PREBUILT)
 
 include $(CLEAR_VARS)
+<<<<<<< HEAD
```
Comment (Shr3ps) [inline rootdir/Android.mk]: o_O you pushed a conflict

**Comment by karthick111** [comment]
Wip don't merge

**Comment by karthick111** [comment]
@skritchz

**Pull request closed without merging**